app/components/sign-up/form.hbs

```diff
@@ -0,0 +1,56 @@
+{{#if @changeset.errors}}
+  <div class="text-red">
+    {{t "signup.error"}}
+  </div>
+{{/if}}
+<form {{action "submit" on="submit"}}>
+  <div>
+    <ValidatedField
+      @type="text"
+      @changeset={{@changeset}}
+      @property="email"
+      @updateProperty={{fn this.updateProperty}}
+      @validateProperty={{fn this.validateProperty}}
```

Comment:
The debouncing doesn't seem to be working with this `ValidatedField` component...may have to convert this one to a regular input just for this case. Username as well.